dotnet/src/Skills/Skills.Web/Bing/BingConnector.cs

```diff
@@ -114,9 +145,16 @@ private sealed class WebPages
         public WebPage[]? Value { get; set; }
     }
 
+    /// <summary>
+    /// A sealed class containing the deserialized response from the Bing Search API.
+    /// </summary>
+    /// <returns>A WebPage object containing the Bing Search API response data.</returns>
+    /// <remarks><b>Note:</b> Complex types are not compatible with the kernel for chaining yet. See <see href="https://github.com/microsoft/semantic-kernel/pull/2053#discussion_r1274782835">PR#2053</see></remarks>
     [SuppressMessage("Performance", "CA1812:Internal class that is apparently never instantiated",
```

Comment:
This suppression shouldn't be needed as-is -- but I think it's better to make this internal for tests.